packages/react-components/src/components/CaptionsBanner.tsx

```diff
@@ -329,17 +329,20 @@ export const CaptionsBanner = (props: CaptionsBannerProps): JSX.Element => {
           .map((caption) => {
             if ('message' in caption) {
               return (
-                <li key={`RealTimeText - ${caption.id}`} className={captionContainerClassName} data-is-focusable={true}>
+                <div
+                  key={`RealTimeText - ${caption.id}`}
+                  className={captionContainerClassName}
+                  data-is-focusable={true}
```

Comment:
this make the content of the div keyboard navigable?

Reply:
yes